python/lsst/ts/watcher/rules/cp_verify_bias_alarm.py

```diff
@@ -0,0 +1,255 @@
+# This file is part of ts_watcher.
+#
+# Developed for Vera C. Rubin Observatory Telescope and Site Systems.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+__all__ = ["CpVerifyBiasAlarm"]
+
+import yaml
+import json
+import lsst.daf.butler as dafButler
+
+from lsst.ts.idl.enums.Watcher import AlarmSeverity
+from lsst.ts import watcher
+
+
+class CpVerifyBiasAlarm(watcher.BaseRule):
```

Comment:
Instead of `CpVerifyBiasAlarm`, let's make it a generic `CpVerifyAlarm`

Reply:
Sounds good. I had it like that initially, and I believe we could make this a general alarm.